### PR TITLE
Change "Parents" to "Parents and Students" on /yourschool

### PR DESCRIPTION
**Pull request by drewsamnick**
This also ads an option to ResourceCard to allow the title to wrap.

### CODE REVIEW

**Comment by drewsamnick** [comment]
[Change "Parents" to Parents and Students"](https://trello.com/c/cvknU6jZ/71-change-parents-to-parents-and-students)

File: apps/src/templates/census2017/YourSchoolResources.jsx [hunk 1]
```diff
@@ -44,11 +44,12 @@ class YourSchoolResources extends Component {
         </div>
         <div style={styles.card}>
           <ResourceCard
-            title={i18n.parents()}
+            title={i18n.parentsAndStudents()}
             description={i18n.yourSchoolParentDesc()}
             buttonText={i18n.yourSchoolParentButton()}
             link={pegasus('/help')}
             isRtl={false}
+            allowWrap={true}
```
Comment (Erin007) [inline apps/src/templates/census2017/YourSchoolResources.jsx:52]: 👍🏼 parameter makes sense, thanks

File: apps/src/templates/studioHomepages/ResourceCard.jsx [hunk 2]
```diff
@@ -11,6 +11,9 @@ const styles = {
     width: 310,
     background: color.teal
   },
+  card_allow_wrap: {
```
Comment (Erin007) [inline apps/src/templates/studioHomepages/ResourceCard.jsx]: these labels are usually camelcase, e.g `cardAllowWrap`

Comment (drewsamnick) [reply apps/src/templates/studioHomepages/ResourceCard.jsx]: Fixed

File: apps/src/templates/studioHomepages/ResourceCard.jsx [hunk 3]
```diff
@@ -23,10 +26,15 @@ const styles = {
...
     width: 260,
     display: 'inline',
+  },
+  titleNoWrap: {
     whiteSpace: 'nowrap',
     textOverflow: 'ellipsis',
     overflow: 'hidden'
   },
+  titleAllowWrap: {
+    lineHeight: '110%'
```
Comment (breville) [inline apps/src/templates/studioHomepages/ResourceCard.jsx]: `1.1` should work here.

**Comment by drewsamnick** [comment]
Sorry, here are images.

Before:
screen shot 2017-11-16 at 2 57 11 pm

After:
screen shot 2017-11-16 at 3 02 15 pm

File: apps/src/templates/studioHomepages/ResourceCard.jsx (hunk 3 above)
Comment (drewsamnick) [reply apps/src/templates/studioHomepages/ResourceCard.jsx]: Updated

**Pull request merged**